.github/workflows/notify-release.yml

```diff
@@ -13,5 +13,5 @@ jobs:
         id: notify
         uses: nearform/github-action-notify-release@main
         with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
+          github-token: ${{ github.token }}
```

Comment:
remove this, since this is using this action!

Reply:
done